packages/flutter/test/painting/noop_codec.dart

```diff
@@ -0,0 +1,27 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+class NoopCodec implements Codec {
```

Comment:
Nit: consider adding an `instance` static member. I don't feel strongly about this either, feel free to skip.

Example:

https://github.com/flutter/flutter/blob/6193615f90a800ff5623373137bd88ea67d5a1d2/packages/flutter/lib/src/foundation/memory_allocations.dart#L123-L128